controllers/dogu_manager.go

```diff
@@ -45,12 +43,7 @@ type DoguManager struct {
 }
 
 // NewDoguManager creates a new instance of DoguManager
-func NewDoguManager(client client.Client, ecosystemClient ecoSystem.EcoSystemV1Alpha1Interface, operatorConfig *config.OperatorConfig, cesRegistry cesregistry.Registry, eventRecorder record.EventRecorder) (*DoguManager, error) {
-	err := validateKeyProvider(cesRegistry.GlobalConfig())
-	if err != nil {
-		return nil, fmt.Errorf("failed to validate key provider: %w", err)
-	}
-
+func NewDoguManager(client client.Client, ecosystemClient ecoSystem.EcoSystemV1Alpha1Interface, operatorConfig *config.OperatorConfig, eventRecorder record.EventRecorder) (*DoguManager, error) {
```

Comment:
🚫 **[checkstyle]** <funlen> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function 'NewDoguManager' is too long (74 > 60)
<!-- __reviewdog__:ChBlZWVlMzJiOGU2MGNkMmRmEgpjaGVja3N0eWxl -->

Reply:
Is see no option to fix this.